x-pack/plugins/canvas/canvas_plugin_src/functions/browser/handlebars_evaluator.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*eslint new-cap: "off"*/
+
+// Handlebars parser doesn't contain types for Visitor so keeping file as vanilla JS
+const Handlebars = require('@handlebars/parser');
+const _ = require('lodash');
+const tinymath = require('@kbn/tinymath');
+
+const isString = (val) => typeof val === 'string';
+function pivotObjectArray(rows, columns) {
+  const columnNames = columns || Object.keys(rows[0]);
+  if (!columnNames.every(isString)) {
+    throw new Error('Columns should be an array of strings');
+  }
+  const columnValues = _.map(columnNames, (name) => _.map(rows, name));
+  return _.zipObject(columnNames, columnValues);
+}
+
+const knownHelpers = {
```

Comment:
If we leave this as an object, I think it'd be worthwhile to freeze it so that a vulnerability elsewhere in Kibana can't alter our expected set of helpers